app/assets/javascripts/main.js.erb

```diff
@@ -131,7 +131,8 @@ $(function() {
           var thingTitles = [];
           $(data).each(function(index, thing) {
             var thingType = thing.species || 'Thing';
-            var thingTitle = thingType + '_' + thing.id;
+            // var thingTitle = thingType + '_' + thing.id; //
+            var thingTitle = ''
```

Comment:
Why are all of the thingTitles now being set to empty strings? Does this impact how the UI works?